Source/ThirdParty/ANGLE/src/libANGLE/validationESEXT.cpp

```diff
@@ -4461,6 +4461,30 @@ bool ValidateLogicOpANGLE(const PrivateState &state,
     return ValidateLogicOpCommon(state, errors, entryPoint, opcodePacked);
 }
 
+// GL_WEBKIT_explicit_resolve_target
+bool ValidateFramebufferResolveRenderbufferWEBKIT(const Context *context,
+                                                  angle::EntryPoint entryPoint,
+                                                  GLenum target,
+                                                  GLenum attachment,
+                                                  GLenum renderbuffertarget,
+                                                  RenderbufferID renderbuffer)
+{
+#if ANGLE_WEBKIT_EXPLICIT_RESOLVE_TARGET_ENABLED
+    if (!context->getExtensions().explicitResolveTargetWEBKIT)
+    {
+        ANGLE_VALIDATION_ERROR(GL_INVALID_OPERATION, kExtensionNotEnabled);
+        return false;
+    }
+
+    // TODO(djg): Check compatibility with default
+    return ValidateFramebufferRenderbufferBase(context, entryPoint, target, attachment,
```

Comment:
In an upstreamed extension this would require a lot more validation, but we're controlling the use of the API and not exposing it publicly.

Reply:
But an attacker could call it with arbitrary arguments..

Reply:
I've added validation of compatibility of the attachments to glCheckFramebufferStatus, since at that point we have both attachments to compare with each other. This validation is the equivalent of the validation performed by `glFramebufferRenderbuffer(...)`